crates/bitwarden/src/commands/projects.rs

```diff
@@ -23,6 +28,37 @@ pub(crate) async fn get_project(
     ProjectResponse::process_response(res, enc)
 }
 
+pub(crate) async fn create_project(
+    client: &mut Client,
+    input: &ProjectCreateRequest,
+) -> Result<ProjectResponse> {
+    let enc = client
+        .get_encryption_settings()
+        .as_ref()
+        .ok_or(Error::VaultLocked)?;
+
+    let org_id = Some(input.organization_id);
+
+    let project = Some(ProjectCreateRequestModel {
+        name: enc.encrypt(input.name.as_bytes(), org_id)?.to_string(),
+    });
+
+    let config = client.get_api_configurations().await;
+    let res = bitwarden_api_api::apis::projects_api::organizations_organization_id_projects_post(
+        &config.api,
+        input.organization_id,
+        project,
+    )
+    .await?;
+
+    let enc = client
+        .get_encryption_settings()
+        .as_ref()
+        .ok_or(Error::VaultLocked)?;
```

Comment:
Isn't `enc` already defined above? I think we can reuse it.

Reply:
Hmm, any recommendations on a better way to do this? By removing it we break one of the borrowing rules. With it removed we have the following error, the compiler delineates the following:
- immutable borrow on line 35/36
- mutable borrow on line 46
- immutable borrow on line 54

`cannot borrow *client as mutable because it is also borrowed as immutable`

Reply:
Discussed with @dani-garcia, addressing this in this PR: https://github.com/bitwarden/sdk/pull/70